internal/pkg/deploy/cloudformation/stack/testdata/workloads/worker-manifest.yml

```diff
@@ -0,0 +1,44 @@
+# The manifest for the "dog-worker" service.
+# Read the full specification for the "Worker Service" type at:
+# https://aws.github.io/copilot-cli/docs/manifest/worker-service/
+
+# Your service name will be used in naming your resources like log groups, ECS services, etc.
+name: dogworker
+type: Worker Service
+
+# Configuration for your containers and service.
+image:
+  # Docker build arguments.
+  build: ./Dockerfile
+
+cpu: 256       # Number of CPU units for the task.
+memory: 512    # Amount of memory in MiB used by the task.
+count: 1       # Number of tasks that should be running in your service.
+exec: true     # Enable running commands in your container.
+
+subscribe:
+  queue:
+    delay: 1s
+    dead_letter:
+          tries: 5
```

Comment:
```suggestion
      tries: 5
```

Reply:
Yes, this template works properly given that subscribers, docker files, etc exist.